packages/cli/internal/pkg/cli/actionable/error.go

```diff
@@ -0,0 +1,38 @@
+package actionable
```

Comment:
Does it make sense to make "actionable" a sub package of clierror? 
Which one conveys intent better in your opinion?
- `import "github.com/aws/amazon-genomics-cli/internal/pkg/cli/actionable"`
- `import "github.com/aws/amazon-genomics-cli/internal/pkg/cli/clierror/actionable"`
- `import "github.com/aws/amazon-genomics-cli/internal/pkg/cli/actionableerror"`